src/sensor_device.cc

```diff
@@ -120,6 +120,10 @@ void Sensor::update (nut::TcpClient &conn, const std::map <std::string, std::str
                         log_debug ("sa: new style dry-contact status, but missing config");
                     }
                 }
+                // workaround for EMP01: state is "open" or "closed"
+                else if (state == "open") {
```

Comment:
the close/closed part is missing, isn't it?

Reply:
no because state is: open/closed, need change only for open state. closed is already ok.